torch/fx/experimental/proxy_tensor.py

```diff
@@ -534,7 +627,7 @@ def wrapped(*proxies):
         )
         out = pytree.tree_map_only(
             (SymInt, SymFloat, SymBool),
-            lambda t: get_proxy_slot(t.node, tracer)(),
+            lambda t: get_proxy_slot(t, tracer)(),
```

Comment:
looks like some sort of refactor

Reply:
This is because we're no longer using the node to hash, but the SymInt itself